cmd/plugin/kubectl/kubectl.go

```diff
@@ -77,7 +77,11 @@ func execToWriter(args []string, writer io.Writer) error {
 		return err
 	}
 
-	go io.Copy(writer, op)
+	go func() {
```

Comment:
I am not sure what we are doing here, but I think that instead of calling this routine we should use cmd.Sysout and write directly to it?

Otherwise, I wouldn't care too much with this plugin and add a //nolint rule on it

Reply:
I think we can just add a //nolint rule here to avoid breaking change.